target/scripts/startup/setup.d/postfix.sh

```diff
@@ -171,7 +171,7 @@ function _setup_SRS
   if [[ -n ${SRS_EXCLUDE_DOMAINS} ]]
   then
     sed -i \
-      "s/^#\?(SRS_EXCLUDE_DOMAINS=).*$/\1=${SRS_EXCLUDE_DOMAINS}/g" \
+      "s/^#\?\(SRS_EXCLUDE_DOMAINS=\).*$/\1${SRS_EXCLUDE_DOMAINS}/g" \
```

Comment:
```suggestion
    sedfile -i -E \
      "s|^#?(SRS_EXCLUDE_DOMAINS=).*|\1${SRS_EXCLUDE_DOMAINS}|" \
```

Reply:
Thanks! Yes, I like this better. Extended re's are also way easier to read.